clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1197,6 +1197,19 @@ If you would like to cite this work, you can use the following BibTeX entry:
 }
 \endverbatim
 
+\section sec_google_common_issues Common Issues
+
+Here we collect common issues that some users might experience when using the Google
+Workload Traces.
+
+- Depending on the Linux distribution used, there might be a threshold on the number of
+  files a single process can open that is set too low (e.g., 1024). This threshold can be
+  increased by setting the nofile item in /etc/security/limits.conf. A low nofile
```

Comment:
The easy local fix that does not require root is to up the soft limit in the currrent shell: e.g., `ulimit -n 8192`. On Ubuntu the default soft limit seems to be 1024, and at least one machine I checked has an empty limits.conf file so it's coming from PAM or some other place. I doubt the hard limit is that low so this local increase should always work.

So I'm suggesting we recommend `ulimit -n 8192` (can put this in .bashrc too for long term fix) and maybe mention in parentheses the limits.conf file or maybe don't mention at all and they can look it up if they want to change things as root.